src/org/labkey/test/components/react/QueryChartPanel.java

```diff
@@ -100,11 +100,13 @@ public static class QueryChartPanelFinder extends WebDriverComponentFinder<Query
     {
         private final QueryGrid _queryGrid;
         private final Locator.XPathLocator _baseLocator = Locator.tagWithClass("div", "chart-panel");
+        private final String _name;
 
-        public QueryChartPanelFinder(WebDriver driver, QueryGrid queryGrid)
+        public QueryChartPanelFinder(WebDriver driver, QueryGrid queryGrid, String name)
         {
             super(driver);
             _queryGrid = queryGrid;
+            _name = name;
```

Comment:
The chart name shouldn't be mandatory. The finder should find all of the charts by default and there should be a `withName` method to specify the name.
`QueryChartPanel` should probably have a `getName` method too.